src/linkml/include_study.yaml

```diff
@@ -160,3 +423,129 @@ enums:
       x01_hakonarson:
         text: x01_hakonarson
         title: X01-Hakonarson
+  enum_researchDomain:
+    name: enum_researchDomain
+    definition_uri: include:enum_researchDomain
+    permissible_values:
+      behavior_and_behavior_mechanisms:
+        text: behavior_and_behavior_mechanisms
+        title: Behavior and Behavior Mechanisms
+        meaning: mesh:D001520
+      congenital_heart_defects:
+        text: congenital_heart_defects
+        title: Congenital Heart Defects
+        meaning: mesh:D006330
+      immune_system_diseases:
+        text: immune_system_diseases
+        title: Immune System Diseases
+        meaning: mesh:D007154
+      hematologic_diseases:
+        text: hematologic_diseases
+        title: Hematologic Diseases
+        meaning: mesh:D006402
+      sleep_wake_disorders:
+        text: sleep_wake_disorders
+        title: Sleep Wake Disorders
+        meaning: mesh:D012893
+      all_co_occurring_conditions:
+        text: all_co_occurring_conditions
+        title: All Co-occurring Conditions
+        meaning: mesh:D013568
+      other:
+        text: other
+        title: Other
+  enum_participantLifespanStage:
+    name: enum_participantLifespanStage
+    definition_uri: include:enum_participantLifespanStage
+    permissible_values:
+      fetal:
+        text: fetal
+        title: Fetal
+      neonatal:
+        text: neonatal
+        title: Neonatal
+        description: 0-28 days old
+      pediatric:
+        text: pediatric
+        title: Pediatric
+        description: Birth-17 years old
+      adult:
+        text: adult
+        title: Adult
+        description: 18+ years old
+  enum_clinicalDataSourceType:
+    #TODO: replace enum_conditionDataSource with this - or consider deleting slot conditionDataSource
+    name: enum_clinicalDataSourceType
+    definition_uri: include:enum_clinicalDataSourceType
+    permissible_values:
+      medical_record:
+        text: medical_record
+        title: Medical Record
+        description: Data obtained directly from medical record
+      investigator_assessment:
+        text: investigator_assessment
+        title: Investigator Assessment
+        description: Data obtained by examination, interview, etc. with investigator
+      participant_or_caregiver_report:
+        text: participant_or_caregiver_report
+        title: Participant or Caregiver Report
+        description: Data obtained from survey, questionnaire, etc.
+      other:
```

Comment:
Why are both `other` and `unknown` listed? Is there somewhere I can read what the use cases are for these?

Reply:
Ideally if we had a form for data contributors, the "Other" field would allow them to write in a response that we could add to the enums.
Unknown = either they don't know, or we don't know and weren't able to find out from contributor